src/rngs/mock.rs

```diff
@@ -75,6 +76,18 @@ impl RngCore for StepRng {
     }
 }
 
+impl SeedableRng for StepRng {
+    type Seed = [u8; 16];
+
+    fn from_seed(seed: Self::Seed) -> Self {
+        let seed_u64: [u64; 2] = transmute!(seed);
```

Comment:
Concern: we should make results portable where (easily enough) possible. `transmute!` presumably does not alter the value, so we would need to use `u64::from_le` (or `u64::from_le_bytes` with [`as_chunks`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.as_chunks)) here. Note that `rand_core::impls::fill_bytes_via_next` uses `to_le_bytes` so this would cancel out with a word-based-generator on a BE platform.